runtime/init.go

```diff
@@ -231,6 +233,27 @@ func (d *Default) InitCNI() error {
 	if d.WithoutCNI {
 		return nil
 	}
+
+	// if interface is "", use master[0] network interface
+	if d.Interface != "" {
```

Comment:
```suggestion
	if interface == "" {
	    using the interface witch bind to master0 IP
	} else {
	    if interface NotIn master0InterfaceList {
	       return err("failed to found xxx nic")
	    }
	}
```
I think the main logic like this is better.